activerecord/lib/active_record/associations.rb

```diff
@@ -1767,7 +1767,8 @@ def has_one(name, scope = nil, **options)
         #   you don't want to have association presence validated, use <tt>optional: true</tt>.
         # [:default]
         #   Provide a callable (i.e. proc or lambda) to specify that the association should
-        #   be initialized with a particular record before validation.
+        #   be initialized with a particular record before validation if it doesn't exist.
+        #   Please note that callable won't be executed if the record exists
```

Comment:
```suggestion
        #   Please note that callable won't be executed if the record exists.
```

Reply:
But, the `if it doesn't exist` should cover this entire line should not it? I think we either should remove this line or remove the `it doesn't exist` from the previous line.

Reply:
Yeah, I felt it was kind of redundant as well. I'm okay with removing it from previous line 😄 👍🏽

Reply:
Done 🙌🏽